python/ml4ir/applications/classification/pipeline.py

```diff
@@ -168,6 +170,31 @@ def get_relevance_dataset(
 
         return relevance_dataset
 
+    def finish(self, job_status, job_info):
+        # This will write the _SUCCESS/_FAILURE files
+        super().finish(job_status, job_info)
+
+        # Create model_bundle.zip depending on user args
+        if self.args.write_tf_model_dir:
+            os.makedirs(self.args.write_tf_model_dir)
```

Comment:
@balikasg I recommend moving all IO to `ml4ir/base/io/file_io` if possible and only keep the function call here.

Reply:
I will do that.